module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
configs comes to mind, e.g. `-sMyVar=123`. I could imagine allowing the user to specify a dictionary of config name/values. Those would then become `-s` flags to the chapel compiler.